File: app/src/main/java/moe/tlaster/openween/core/model/Url/AnnotationModel.java
package moe.tlaster.openween.core.model.url;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Tlaster on 2016/9/2.
 */
public class AnnotationModel {
    @SerializedName("object")
    private ObjectModel mItem;

    public ObjectModel getItem() {
        return mItem;
    }

    public void setItem(ObjectModel item) {
        mItem = item;
    }
}

File: app/src/main/java/moe/tlaster/openween/core/model/url/AnnotationModel.kt
package moe.tlaster.openween.core.model.url

import com.fasterxml.jackson.annotation.JsonProperty


/**
 * Created by Tlaster on 2016/9/2.
 */
class AnnotationModel {
    @field:JsonProperty("object")
    var item: ObjectModel? = null
}
